app/boards/shields/rebound/Kconfig.defconfig

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if SHIELD_REBOUND_V4
+
+config ZMK_KEYBOARD_NAME
+    default "Rebound_v4"
```

```suggestion
    default "Rebound v4"
```
We can have spaces in here. What do you think of that instead?